core/dronecore.h

```diff
@@ -48,26 +47,45 @@ class DroneCore
      * Supports connection: Serial, TCP or UDP.
      * Connection URL format should be:
      * - UDP - udp://[Bind_host][:Bind_port]
-     * - TCP - tcp://[Server_host][:Server_port]
-     * - Serial - serial://[Dev_Node][:Baudrate]
+     * - TCP - tcp://[Remote_host][:Remote_port]
+     * - Serial - serial://Dev_Node[:Baudrate]
      *
-     * Default URL : udp://:14540.
-     * - Default Bind host IP is localhost(127.0.0.1)
-     *
-     * @warning Serial connections are not supported on Windows (they are supported on Linux and macOS).
+     * Default URL : udp://0.0.0.0:14540.
+     * - Default Bind host IP is any local interface (0.0.0.0)
      *
      * @param connection_url connection URL string.
      * @return The result of adding the connection.
      */
-    ConnectionResult add_any_connection(const std::string &connection_url = DEFAULT_UDP_CONNECTION_URL);
+    ConnectionResult add_any_connection(const std::string &connection_url);
+
+    /**
+     * @brief Adds a UDP connection to the specified port number on 0.0.0.0 on any local interface.
```

Comment:
Does "any local interface" mean any network? (ie I am wondering what this means as opposed to if you did not have this text)

Reply:
"0.0.0.0" does means "listen on any interface". So "on any local interface" here does not add information, but explicits what "0.0.0.0" means.

Here it is [from Wikipedia](https://en.wikipedia.org/wiki/0.0.0.0):

> In the context of servers, 0.0.0.0 can mean "all IPv4 addresses on the local machine". If a host has two IP addresses, 192.168.1.1 and 10.1.2.1, and a server running on the host is configured to listen on 0.0.0.0, it will be reachable at both of those IP addresses.